sources/packages/web/src/types/contracts/student/ApplicationContract.ts

```diff
@@ -192,6 +188,33 @@ export interface Assessment {
   otherAllowableCost: number;
 }
 
+/**
+ * DTO for PartTime assessment payload.
+ */
+export interface PartTimeAssessment {
+  weeks: number;
```

Comment:
There a couple of properties that are shared between FullTime/ParTime, maybe it would be good to have the common ones in a base class. For the ones that are not the same for PartTime, is it possible that they have the same meaning as the other one in FullTime?

Reply:
Sure, that is right, I have created a base assessment interface.